TDD/TDD1_BerlinClock/src/main/java/logic/impl/Converter.java

```diff
@@ -0,0 +1,72 @@
+package logic.impl;
+
+public class Converter {
```

Comment:
It's good that you excluded Coverter class, but, please rethink what is should do.
It looks like some of it's concepts are shared with Berlin Clock class.
I would suggest to rethink organization of this code.
I mean either it converts types (String to int etc.) or it is a logical engine of berlin clock (converts some specific value of int to a BerlinClock output).
Those two functionalities are spearate things, and code structure should reflect that.